app/src/main/java/com/github/pockethub/ui/WebView.java

```diff
@@ -17,12 +17,15 @@
 
 import android.content.Context;
 import android.util.AttributeSet;
+import android.view.MotionEvent;
 
 /**
  * Web view extension with scrolling fixes
  */
 public class WebView extends android.webkit.WebView {
```

Comment:
Is this class used anywhere else? Might want to make a specific subclass for the README/GitHub specific places